dot/core/service.go

```diff
@@ -137,8 +137,8 @@ func NewService(cfg *Config) (*Service, error) {
 		// create a new BABE session
 		bs, err := babe.NewSession(bsConfig)
 		if err != nil {
-			log.Error("[core] could not start babe session", "error", err)
 			srv.isBabeAuthority = false
+			log.Error("[core] could not start babe session", "error", err)
```

Comment:
is this the new default prefix to use now for any method inside core ? [core] ?

Reply:
yeah, i think [package name] seems pretty standard for logging. there wasn't much consistency before, so i think if everyone is okay with this logging style, we should update the logs.

Reply:
yes but only for core or all packages ? if eg: core/types the log would be [types] or [core] ?

Reply:
good point, i think we would have to put [core/types] since there are multiple types packages.  let's discuss this on our next call